tests/framework/base/ModuleTest.php

```
@@ -36,214 +51,698 @@ public function testTrueParentModule(): void
+        $module = new TestModule('test');
+        $module->actionMap = [
+            'noop' => GreetAction::class,
+        ];
+
+        Event::on(
+            GreetAction::class,
+            Controller::EVENT_BEFORE_ACTION,
+            static function (ActionEvent $event): void {
+                $event->isValid = false;
+            },
+        );
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\yii\base\Event' in method 'testRunStandaloneActionShortCircuitsWhenBeforeActionEventIsInvalidated'. (undefined)

(StaticAccess)